src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/SocketsHttpHandler.cs

```diff
@@ -173,6 +172,13 @@ public HttpKeepAlivePingPolicy KeepAlivePingPolicy
             set => throw new PlatformNotSupportedException();
         }
 
+        [CLSCompliant(false)]
```

Comment:
>CLSCompliant [](http://example.com/codeflow?start=9&length=12)

Why this is false CLS compliance? just curious :-)

Reply:
The `DiagnosticSource` assembly is marked as `<CLSCompliant>false</CLSCompliant>`, which makes all its types (including `TextMapPropagator`) non-compliant.

https://github.com/dotnet/runtime/blob/21c251623d45e2a753cd687c0551ec327f837d51/src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.csproj#L4

Reply:
Thanks for the info. did you get failure without adding this attribute?

Reply:
`error CS3003: Type of 'TextMapPropagator' is not CLS-compliant`

Reply:
thanks!